Secp256k1.Net/Interop.cs

```diff
@@ -118,6 +118,21 @@ namespace Secp256k1Net
         void* sigin     // const secp256k1_ecdsa_signature* sigin
     );
 
+    /// <summary>
+    /// Parse a DER ECDSA signature
```

Comment:
Can you add the full text from the original header file here? https://github.com/bitcoin-core/secp256k1/blob/e96901a4b951dfb6336114d223151dd0f5ffb89b/include/secp256k1.h#L359

Thank you!

Reply:
I have added the comment as requested